src/lib/mongoBackend/safeMongo.cpp

```diff
@@ -234,8 +234,9 @@ bool moreSafe(const std::auto_ptr<DBClientCursor>& cursor)
 * nextSafeOrError -
 *
 */
-bool nextSafeOrError(const std::auto_ptr<DBClientCursor>& cursor, BSONObj* r, std::string* err)
+bool nextSafeOrError(const std::auto_ptr<DBClientCursor>& cursor, BSONObj* r, std::string* err, const std::string& caller, int line)
 {
+  // FIXME P4: use line in the trace (just "+ line" doesn't work ;)
```

Comment:
Not so important for this PR, but if somebody have suggestions, please, go ahead with the code.